src/main/java/org/myrobotlab/service/GoogleSearch.java

```diff
@@ -126,7 +125,7 @@ public SearchResults search(String searchText) throws IOException {
       String encodedSearch = URLEncoder.encode(searchText, "UTF-8");
 
       // not sure if locale is supported tag probably is ....
-      String request = "https://google.com/search?lr=lang_" + locale.getTag() + "&q=" + encodedSearch + "&aqs=chrome..69i57.5547j0j7&sourceid=chrome&ie=UTF-8";
+      String request = "https://google.com/search?lr=lang_" + Runtime.getInstance().getLocaleTag() + "&q=" + encodedSearch + "&aqs=chrome..69i57.5547j0j7&sourceid=chrome&ie=UTF-8";

```

Comment:
This should be using the locale of the google search service.. not runtime.

Reply:
nice catch ! .. thanks ..
fixed